src/main/java/com/ning/billing/recurly/model/Account.java

```diff
@@ -605,6 +618,9 @@ public boolean equals(final Object o) {
         if (parentAccountCode != null ? !parentAccountCode.equals(account.parentAccountCode) : account.parentAccountCode != null) {
             return false;
         }
+        if (billTo != null ? !billTo.equals(account.billTo) : account.billTo != null) {
```

Comment:
Seems like this is equal to `Objects.equals(billTo, account.billTo)`

Reply:
Yes, but I'm following the style of the existing code.